### PR TITLE
PostItem: Add action counts to the right of status

### PR DESCRIPTION
**Pull request by jonathansadowski**
This PR adds action counts (# of likes, views, and comments) to the right of the post status.

Before:
image

After:
image

Note: this counts are just informational, they will be updated to link to the appropriate section in a future PR.

### CODE REVIEW

File: client/my-sites/post-type-list/post-action-counts/index.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+		);
+	}
+
+	renderViewCount() {
+		const { viewCount: count, numberFormat, translate } = this.props;
+
+		return (
+			<li>
+				{ translate(
+					'%(count)s view',
```
Comment (a8ci18n) [inline client/my-sites/post-type-list/post-action-counts/index.jsx]: Hi! I've found a possible matching string that has already been translated **22** times:
`translate( '%(count)s Total View', '%(count)s Total Views' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/post-type-list/post-action-counts/index.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+		);
+	}
+
+	renderLikeCount() {
+		const { likeCount: count, numberFormat, translate } = this.props;
+
+		return (
+			<li>
+				{ translate(
+					'%(count)s like',
```
Comment (a8ci18n) [inline client/my-sites/post-type-list/post-action-counts/index.jsx]: Hi! I've found a possible matching string that has already been translated **23** times:
`translate( '%(count)s Like', '%(count)s Likes' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/post-type-list/post-action-counts)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/post-type-list/post-action-counts/index.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+		globalId: PropTypes.string,
+	};
+
+	renderCommentCount() {
+		const { commentCount: count, numberFormat, translate } = this.props;
+
+		return (
+			<li>
+				{ translate(
+					'%(count)s comment',
```
Comment (a8ci18n) [inline client/my-sites/post-type-list/post-action-counts/index.jsx]: Hi! I've found a possible matching string that has already been translated **22** times:
`translate( '%(count)s Comment', '%(count)s Comments' )` *ES Score: 18*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/post-type-list/post-action-counts)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/post-type-list/post-action-counts/index.jsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+		);
+	}
+
+	renderViewCount() {
+		const { viewCount: count, numberFormat, translate } = this.props;
+
+		return (
+			<li>
+				{ translate(
+					'%(count)s View',
```
Comment (a8ci18n) [inline client/my-sites/post-type-list/post-action-counts/index.jsx]: Hi! I've found a possible matching string that has already been translated **22** times:
`translate( '%(count)s Total View', '%(count)s Total Views' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by jonathansadowski** [comment]
Thanks for the review!

**Pull request merged**